test/testio.cpp

```diff
@@ -81,28 +81,34 @@ class TestIO : public TestFixture {
         TEST_CASE(testParameterPack); // #11289
     }
 
+    struct CheckOptions
+    {
+          CheckOptions() = default;
+          bool inconclusive = false;
+          bool portability = false;
+          Platform::Type platform = Platform::Type::Unspecified;
+          bool onlyFormatStr = false;
+          bool cpp = true;
+    };
+
 #define check(...) check_(__FILE__, __LINE__, __VA_ARGS__)
-    void check_(const char* file, int line, const char* code, bool inconclusive = false, bool portability = false, Platform::Type platform = Platform::Type::Unspecified, bool onlyFormatStr = false, bool cpp = true) {
+    void check_(const char* file, int line, const char* code, const CheckOptions& options = make_default_obj()) {
         // Clear the error buffer..
         errout.str("");
 
         // TODO: using dedicated Settings (i.e. copying it) object causes major slowdown
-        settings1.severity.clear();
```

Comment:
This removed the default `Serverity::error`. Fortunately it was not used in this test.